server/src/api/router.go

```diff
@@ -130,7 +137,7 @@ func (s *Server) publicRoutes(r chi.Router) chi.Router {
 		r.Post("/feedback", s.createFeedback)
 		r.Route("/login", func(r chi.Router) {
 			r.Delete("/", s.logout)
-			r.Post("/anonymous", s.signInAnonymously)
+			r.With(s.AnonymousLoginEnabledContext).Post("/anonymous", s.signInAnonymously)
```

Comment:
If I understand this correctly, you are not able to login with a anonymous account. But you would still be able to join a board if you were already logged in.

Reply:
Yes exactly. While this seems sloppy on my part it was actually intended; since usually this is an option set at the very start and likely only once, not changed frequently. Since this is built with #3760 in mind, I think it's more likely to be set at the very beginning and never changed.  
The option also titled "enable anonymous login", so imo it isn't unclear that already logged in users still may be used. 

However, I am open to discuss renaming the option to something like "enableAnonymousAccountCreation" or disabling the route for anon users alltogether.